devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy

```diff
@@ -183,5 +183,5 @@ def configure() {
 
 	// temperature minReportTime 30 seconds, maxReportTime 5 min. Reporting interval if no activity
 	// battery minReport 30 seconds, maxReportTime 6 hrs by default
-	return refresh() + zigbee.batteryConfig() + zigbee.temperatureConfig(30, 300) // send refresh cmds as part of config
+	return refresh() + zigbee.iasZoneConfig(60, 300) + zigbee.batteryConfig() + zigbee.temperatureConfig(120, 300) // send refresh cmds as part of config
```

Comment:
The minReportingInterval can stay the same for temp, it's the max that matters.  I would leave the min at 30, but max up to 600.  @tpmanley that work for you?

Reply:
I think you could go even higher with the max for temp like 3600 (an hour) seems good to me. It's going to get pushed immediately if it changes and I don't see much benefit of sending it more frequently.